components/common/Footer.tsx

```diff
@@ -1,7 +1,114 @@
+import facebook from "@/public/assets/images/landing-page/Facebook.svg";
```

Comment:
add an index to the folder where thesse images are located and export them 

it is neater to have 

`import {twitter, youtube, ...} from '@public/assets/images/landing-page'`